### PR TITLE
Statistic

### PR DESCRIPTION
**Pull request by Qwe1999**

### CODE REVIEW

**Review by clevercattv — COMMENTED** [review]
pull request to master...

File: src/main/java/com/softserve/academy/event/entity/SurveyAnswer.java [hunk 1]
```diff
@@ -17,6 +17,9 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
+@org.hibernate.annotations.NamedQuery(name="SurveyAnswer.findByQuestionId",
+        query="from SurveyAnswer where questionId = :questionId ")
+
```
Comment (clevercattv) [inline src/main/java/com/softserve/academy/event/entity/SurveyAnswer.java:22]: import org.hibernate.annotations.NamedQuery; - ?

File: src/main/java/com/softserve/academy/event/dto/QuestionStatisticDTO.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    public String[] getAnswers() {
+        return answers;
+    }
+
+    public void setAnswers(String[] answers) {
+        this.answers = answers;
+    }
+}
```
Comment (clevercattv) [inline src/main/java/com/softserve/academy/event/dto/QuestionStatisticDTO.java:53]: Use lombok : 
@Getter
@Setter

File: src/main/resources/database.properties [hunk 3]
```diff
@@ -1,7 +1,7 @@
 hibernate.connection.driver_class=org.postgresql.Driver
-hibernate.connection.url=jdbc:postgresql://localhost:5432/softserve_event_table
+hibernate.connection.url=jdbc:postgresql://localhost:5432/survey
 hibernate.connection.username=postgres
-hibernate.connection.password=root
+hibernate.connection.password=admin
```
Comment (clevercattv) [inline src/main/resources/database.properties:4]: seriously?

File: src/main/java/com/softserve/academy/event/dto/QuestionStatisticDTO.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            QuestionStatisticDTO dto = new QuestionStatisticDTO();
+            dto.answers = mapper.readValue(surveyQuestion.getAnswers(),String[].class);
+            dto.id = surveyQuestion.getId();
+            dto.question = surveyQuestion.getQuestion();
+            return dto;
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
```
Comment (clevercattv) [inline src/main/java/com/softserve/academy/event/dto/QuestionStatisticDTO.java:28]: For mapping SurveyQuestion to QuestionStatisticDTO you can create interface mapper.
See : service.mapper.QuestionMapper for example.

**Review by clevercattv — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserve/academy/event/entity/SurveyAnswer.java (hunk 1 above)
Comment (Qwe1999) [reply src/main/java/com/softserve/academy/event/entity/SurveyAnswer.java:22]: fixed

**Pull request closed without merging**